src/main/java/racingcar/RandomNumMaker.java

```diff
@@ -0,0 +1,10 @@
+package racingcar;
+
+import mallang.missionutils.Randoms;
+
+public class RandomNumMaker {
+
+    public int RandomNum() {
```

Comment:
RandomNum 메소드명도 camel case를 지키지 않고 있네요!
```suggestion
    public int randomNum() {
```